recipe/meta.yaml

```
@@ -34,95 +45,147 @@ build:
 requirements:
   build:
     - cmake
+    - ninja
     - make                    # [unix]
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
+    - {{ compiler('cxx') }}   # [not linux]
+    - {{ compiler('c') }}     # [not linux]
+    - clangxx                 # [linux]
+    - clang                   # [linux]
     - python                  # [not osx]
     - patch                   # [unix]
     - m2-patch                # [win]
+    - llvm
   host:
-    - clangdev {{ version }}  # [not osx]
-    - llvmdev {{ version }}   # [not osx]
+    - clangdev {{ bootstrap_version }}  # [not osx]
+    - llvm {{ bootstrap_version }}      # [not osx]
 
 outputs:
-  - name: libcxx
+
+  - name: libcxx-devel
     files:
-      - include/c++
-      - Library/include/c++
-      - lib/libc++.*
-      - lib/libc++fs.*
-      - lib/libc++experimental.*
-      - Library/bin/c++*.dll
-      - Library/lib/c++*.lib
-      - Library/lib/libc++*.lib
+      - include/c++                 # [unix]
+      - Library/include/c++         # [win]
```

```suggestion
      - include/c++/v1                   # [unix]
      - lib/libc++.${SHLIB_EXT}          # [unix]
      - lib/libc++.a                     # [unix]
      - lib/libc++experimental.a         # [unix]
      - "share/licenses/libc++-devel/*"  # [unix]
```
These are the paths that should be included based on the contents of the only other `libc++-devel` package I could find on a RHEL-ish distro. I am not sure if we're building all of these but I went looking because I would expect a `-devel` to include the generic symlink for the main lib, `libc++.so` in this case. I would also expect these to be prefixed with `usr/` but maybe conda-build is smart enough to just match any path to these as subpaths.

https://www.rpmfind.net/linux/RPM/opensuse/15.6/x86_64/libc++-devel-17.0.6-150600.1.33.x86_64.html